tests/System.Memory.Polyfill.Tests/System.Memory.Polyfill.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <TargetFrameworks>net46;netcoreapp2.1</TargetFrameworks>
+    <AssemblyOriginatorKeyFile>../../tools/test_key.snk</AssemblyOriginatorKeyFile>
```

Comment:
All the properties from here on look unnecessary for a test project.

Reply:
yeah, I just copied them from some other test, but I will remove/simplify. Thanks.